litigious-liberators/profile_creation/quetoscore.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
You use YAML elsewhere, this could probably be YAML. What is the file name as well?